server/db/recreate-schema.sql

```diff
@@ -13,6 +18,8 @@ CREATE TABLE users (
   password VARCHAR(200) NOT NULL
 );
 
+
+
```

Comment:
Could you delete this extra space? Try not to add random space in files :slightly_smiling_face:.